SobokSobok/SobokSobok/Data/Model/SignUpDataModel.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  SignUpDataModel.swift
+//  SobokSobok
+//
+//  Created by 김선오 on 2022/01/18.
+//
+
+import UIKit
+
+struct SignUpDataModel {
+    var email: String?
+    var password: String?
+    var name: String?
+}
+
+extension SignUpDataModel {
+    static var shared: SignUpDataModel = SignUpDataModel(email: "", password: "", name: "")
```

Comment:
다음과 같이 작성한 이유가 있을까요?
```suggestion
    static let shared: SignUpDataModel = SignUpDataModel()
```

Reply:
7b8fae7 반영했습니다